sdks/python/apache_beam/ml/inference/base_test.py

```diff
@@ -339,6 +375,79 @@ def validate_inference_args(
     third_party_model_handler.batch_elements_kwargs()
     third_party_model_handler.validate_inference_args({})
 
+  def test_run_inference_prediction_result_with_model_id(self):
+    examples = [1, 5, 3, 10]
+    expected = [
+        base.PredictionResult(
+            example=example,
+            inference=example + 1,
+            model_id='fake_model_id_default') for example in examples
+    ]
+    with TestPipeline() as pipeline:
+      pcoll = pipeline | 'start' >> beam.Create(examples)
+      actual = pcoll | base.RunInference(
+          FakeModelHandlerReturnsPredictionResult())
+      assert_that(actual, equal_to(expected), label='assert:inferences')
+
+  @pytest.mark.it_postcommit
+  def test_run_inference_prediction_result_with_side_input(self):
```

Comment:
Note to self: I ran out of time to review this test, but don't want to block the rest of the feedback on this. I'll take another look at this when I have time.